src/Uno.Templates/content/unoapp/MyExtensionsApp._1/App.blank.cs

```diff
@@ -3,25 +3,25 @@ namespace MyExtensionsApp._1;
 
 public class App : Application
 {
-	public static Window? _window;
+	public static Window? AppWindow { get; private set; }
```

Comment:
Why is this public?
Why is this a static?

Reply:
I think it is because making it `public static` makes it a bit easier to access - because `Application.Current` is just a plain `Microsoft.UI.Xaml.Application` so you would have to do `((App)Application.Current).MainWindow`, this way you can do just `App.MainWindow`.